### PR TITLE
Define requirements in one place only

### PR DESCRIPTION
**Pull request by ionicsolutions**

### CODE REVIEW

File: bin/req_packages_anovos_py_3_7_x.sh [hunk 1]
```diff
@@ -1,19 +1,5 @@
...
-sudo pip-3.7 install varclushi
-sudo pip-3.7 install soupsieve
-sudo pip-3.7 install datapane==0.12.0
-sudo pip-3.7 install plotly==5.2.1
-sudo pip-3.7 install fsspec
-sudo pip-3.7 install s3fs
-sudo pip-3.7 install --no-deps --force-reinstall matplotlib==3.4.3
\ No newline at end of file
+sudo pip-3.7 install -r ../requirements.txt
+sudo pip-3.7 install --no-deps --force-reinstall matplotlib==3.4.3
```
Comment (ionicsolutions) [inline bin/req_packages_anovos_py_3_7_x.sh:5]: @VishnuGowthem Do you know why this is necessary?

File: bin/req_packages_anovos_py_3_7_x.sh [hunk 2]
```diff
@@ -1,19 +1,5 @@
 sudo pip-3.7 install --upgrade cython
 sudo pip-3.7 install --upgrade pip --user
 sudo pip-3.7 install --upgrade setuptools --user
```
Comment (ionicsolutions) [inline bin/req_packages_anovos_py_3_7_x.sh:3]: @VishnuGowthem Do you know why this is necessary?

**Comment by ionicsolutions** [comment]
@VishnuGowthem I could not see where these scripts are used. Do they belong to some cloud setup?

File: bin/req_packages_anovos_py_3_8_x.sh [hunk 3]
```diff
@@ -1,19 +1,4 @@
-sudo pip3 install --upgrade cython
```
Comment (ionicsolutions) [inline bin/req_packages_anovos_py_3_8_x.sh:1]: @VishnuGowthem Do you happen to know why there is a script for Python 3.8 when we use 3.7 for Spark 2.4.x and 3.9 for Spark 3.x during testing?

**Comment by VishnuGowthem** [comment]
> @VishnuGowthem I could not see where these scripts are used. Do they belong to some cloud setup?

These shell scripts have been used as bootstrap files when starting up AWS EMR clusters, with different versions for different EMR versions.
For example, AWS EMR versions < 5.30, EMR has python 2.7 as system default and spark 2.4.x but support python3 so including pip-3.7 specifically picks up python3
For AWS EMR version >= 5.30 < 6.x, EMR has python3 as system default and supports spark 2.4.x hence just installing pip3 works.
Will have this documented and name files accordingly

File: bin/req_packages_anovos_py_3_7_x.sh (hunk 2 above)
Comment (VishnuGowthem) [reply bin/req_packages_anovos_py_3_7_x.sh:3]: specifically to pick up python 3.7 when 2.7 is EMR default, so this is needed. But file name could be maybe EMR version wise and

**Comment by ionicsolutions** [comment]
> Will have this documented and name files accordingly

Thanks! I think it's a good opportunity to consider how a well-maintained collection of scripts to run on various cloud providers/resources could look like.

**Comment by kajanansangar** [comment]
@ionicsolutions @VishnuGowthem is this ready for PR?

**Comment by ionicsolutions** [comment]
@kajanansangar No, @VishnuGowthem will clean up the naming etc. of the various cloud scripts so that we are well-prepared to add additional ones e.g. for Azure

**Comment by kajanansangar** [comment]
> @kajanansangar No, @VishnuGowthem will clean up the naming etc. of the various cloud scripts so that we are well-prepared to add additional ones e.g. for Azure

Noted @ionicsolutions @VishnuGowthem let's expedite this.

**Pull request merged**